DESCRIPTION

```diff
@@ -30,7 +31,8 @@ Imports:
     stringr,
     utils,
     edgeR,
-    limma  (>= 3.38)
+    limma  (>= 3.38),
+    htmlwidgets
```

Comment:
a curios question :), is not  htmlwidgets already imported with canvasXpress?

Reply:
To avoid this message  _'::' or ':::' import not declared from: ‘htmlwidgets’_.